src/acachecontrol/acachecontrol.py

```diff
@@ -27,6 +32,15 @@ def get(self, url, *, allow_redirects=True, **params):
     def post(self, url, *, data, **params):
         return self.request("POST", url, data=data, **params)
 
+    def put(self, url, *, data, **params):
```

Comment:
I believe that `put` is not cacheable: https://stackoverflow.com/a/107450/13858551
We need to check other methods (verbs).

Reply:
Agree, let met check it and how it is done in CacheControl lib. But, we can set in method self.request any verb, thus this handling should be done in other place.